### ISSUE TITLE
Disposals System [Gladius]

### ISSUE THREAD

**Issue by Bobbanz1** [BeeStation/NSV13#1182]
Since there's no Map Issue with the label Gladius I'm going to put this here as a bug instead.

Trying to ship something through the Gladius's disposal system will always fail and return the package directly back to cargo.

**Comment by SerynEngi** [comment]
I'll take a look at it and correct it in a PR. Don't expect it this weekend, if its anything other than a simple fix, it will take a bit for me to remap parts of the system.

**Comment by SerynEngi** [comment]
Alright. Looked at it. Going to take a minute, it doesn't look like there are any output chutes anywhere other than cargo, which is why it loops back. The fact that it loops back is important information. Before I rebuild sections of it, do you get trash returning through? Based on the location of disposals, this should happen.

**Comment by Koekiegast** [comment]
I'll ad onto this since it's also a disposals issue, the chute in the engineering equipment room just leads to an unfinished pipe section in the engineering lobby.

**Comment by SerynEngi** [comment]
I have corrected the mapping side of it. The way it is set up now, anything going to a major department will end up at the central shipping outlet (for example, xenobiology tagged things will end up in the same place that science dept things will). 

However, in the process, a new bug was discovered. Things tagged with the destination tagger do NOT keep their labels in between Z-levels. I have deferred my map corrections until such time as this coding issue is corrected.

**Comment by AU518987077** [comment]
can confirm above report, i was there doing the testing with seryn on my box.

**Comment by SerynEngi** [comment]
> I'll ad onto this since it's also a disposals issue, the chute in the engineering equipment room just leads to an unfinished pipe section in the engineering lobby.

In reference to that specifically, it has been corrected. There were a couple other departments just like that. In the case of engineering specifically, there are two outputs and two inputs into the line. Engineering lobby, which will receive CE office/ Engineering tagged items, and atmospherics, which will only receive atmospherics tagged items. Right now, those are the only ones that will work, other than disposals proper, as they are on the lower deck. 

Other departments outputs are as follows: 
Cook (output only) receives: Kitchen
Medbay (input/output) receives: Medbay, CMO office, Chemistry, Genetics/ and Virology
Science (input/output) receives: Research, RD Office, Robotics, Toxins, Xenobiology
Security (Output only) receives: Security, HOS Office, Law Office, Detectives Office
Cargo (input/output) receives: Cargo, Quartermaster. (untagged/incorrectly labeled non-waste items)

Aside from munitions (which has its own separate thing that I did not tamper with, as it appeared about as functional as it needed) if its not listed here, it is not included, and will return to cargo (when the bug is fixed).
